app/src/main/java/org/wikipedia/page/PageFragment.java

```diff
@@ -430,7 +429,7 @@ public void onActivityCreated(Bundle savedInstanceState) {
             new LongPressHandler(webView, HistoryEntry.SOURCE_INTERNAL_LINK, new PageContainerLongPressHandler(this));
         }
 
-        pageFragmentLoadState.setUp(model, this, webView, bridge, leadImagesHandler, getCurrentTab());
+        pageFragmentLoadState = new PageFragmentLoadState(model, this, webView, bridge, leadImagesHandler, getCurrentTab());
```

Comment:
Just for safety should we just do this onCreate itself?

Reply:
If we do this in `onCreate`, the `bridge`, `leadImagesHandler` will be `null`.